components/console/logger.rst

```diff
@@ -85,6 +85,8 @@ through the second parameter of the :class:`Symfony\\Component\\Console\\Console
 constructor::
 
     // ...
+    use Psr\Log\LogLevel;
+    // ...
```

Comment:
I think we should just move the `use` statement before the existing placeholder. Then can save the second one.